code/game/objects/items/toys/toys.dm

```diff
@@ -481,6 +481,8 @@
 
 
 /obj/item/toy/beach_ball/basketball/attack_alien(mob/living/carbon/xenomorph/user)
+	if(!CONFIG_GET(flag/fun_allowed))
```

Comment:
putting too many things on this config flag imo

Reply:
It was and already is there, but on the holoball variant. This is not being added.